scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,115 @@
+  fi
+}
+
+SMALL_INSTALL=false
+
+case "${1:-}" in
+    -h|--help)
+        echo -e "Usage:\n\tquick-helm-install.sh [options]"
+        echo " "
+        echo "Installs StackRox via Helm charts."
+        echo " "
+        echo "options:"
+        echo "-h, --help                show brief help"
+        echo "-s, --small               reduce StackRox resource requirements for small clusters"
+        exit 0
+        ;;
+    -s|--small)
+        SMALL_INSTALL=true
+        ;;
+esac
+
+echo "Adding the stackrox/helm-charts/opensource repository to Helm."
+
+helm repo add stackrox https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+
+echo "Generating STACKROX_ADMIN_PASSWORD"
+
+STACKROX_ADMIN_PASSWORD="$(openssl rand -base64 20 | tr -d '/=+')"
+
+echo "Installing stackrox-central-services"
+
+installflags=()
+if [[ "$SMALL_INSTALL" == "true" ]]; then
+    installflags+=('--set' 'central.resources.requests.memory=1Gi')
+    installflags+=('--set' 'central.resources.requests.cpu=1')
+    installflags+=('--set' 'central.resources.limits.memory=4Gi')
+    installflags+=('--set' 'central.resources.limits.cpu=1')
+    installflags+=('--set' 'scanner.autoscaling.disable=true')
+    installflags+=('--set' 'scanner.replicas=1')
+    installflags+=('--set' 'scanner.resources.requests.memory=500Mi')
+    installflags+=('--set' 'scanner.resources.requests.cpu=500m')
+    installflags+=('--set' 'scanner.resources.limits.memory=2500Mi')
+    installflags+=('--set' 'scanner.resources.limits.cpu=2000m')
+fi
+
+helm install -n stackrox --create-namespace stackrox-central-services stackrox/stackrox-central-services \
+ --set central.adminPassword.value="${STACKROX_ADMIN_PASSWORD}" \
+ "${installflags[@]}"
+
+kubectl -n stackrox rollout status deploy/central --timeout=3m
+
+echo "Setting up central port-forward"
+
+kubectl -n stackrox port-forward deploy/central --pod-running-timeout=1m0s 8000:8443 > /dev/null 2>&1 &
+
+echo "Generating an init bundle with stackrox-secured-cluster-services provisioning secrets"
+
+kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
+  --password "${STACKROX_ADMIN_PASSWORD}" \
+  central init-bundles generate stackrox-init-bundle --output - 1> stackrox-init-bundle.yaml
+
+installflags=()
+if [[ "$SMALL_INSTALL" == "true" ]]; then
+    installflags+=('--set' 'sensor.resources.requests.memory=500Mi')
+    installflags+=('--set' 'sensor.resources.requests.cpu=500m')
+    installflags+=('--set' 'sensor.resources.limits.memory=500Mi')
+    installflags+=('--set' 'sensor.resources.limits.cpu=500m')
+fi
+
+echo "Installing stackrox-secured-cluster-services"
+
+helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services \
+ -f stackrox-init-bundle.yaml --set clusterName="my-secured-cluster" \
+ "${installflags[@]}"
+
+echo "Logging into StackRox in the browser"
+
+logmein
+
+echo -e "
+\033[1;31mStackRox is now installed!\033[0m
```

Comment:
Did you intend that this is printed red or green?
This is what I see
![image](https://user-images.githubusercontent.com/537715/187867103-14121d24-56c6-4c92-ae1f-bc153d255c04.png)

Reply:
Yes, It's to make it more celebratory! Do you dislike it?

Reply:
It is ... ok. I'm more used to see red as error or warning color, but as we are **Red**-hatters, showing this in red color is ok.